src/whylogs/viewer/templates/index-hbs-cdn-all-in-jupyter-distribution-chart.html

```diff
@@ -268,7 +268,7 @@ <h1 class="no-responsive__title">Hold on! :)</h1>
             $(document).ready(() => 
               $(".desktop-content").html(`
                 <p style="height: ${$(window).height()}px" class="error-message"> 
-                  Something went wrong. Please try again.
+                     Histogram failed. Check variables are supported.
```

Comment:
I think "variables are supported" still seems a bit vague.
Can we ask for numerical, like:
"Histogram Failed. Features must be of numerical type."
?